pdf/src/campPrint/Responsibles.vue

```diff
@@ -14,9 +14,12 @@
       </Text>
     </View>
   </View>
-  <Text v-else>
-    {{ responsibles }}
-  </Text>
+  <View v-else style="display: flex; flex-direction: row; flex-wrap: wrap">
+    <Text v-for="(responsible, index) in responsibles"
+      >{{ responsible
+      }}<template v-if="index + 1 < responsibles.length">, </template></Text
+    >
```

Comment:
This looks better in my opinion:
```suggestion
<Text v-for="(responsible, index) in responsibles">
{{ responsible}}
<template v-if="index + 1 < responsibles.length">, </template>
</Text>
```

Reply:
Yes from a dev authoring standpoint, I agree, but the outcome is different. I don't want spaces around the names ;)

Reply:
oh, you implemented the .join(', ') with Text elements?
And newlines lead to spaces in "react"-pdf rendering?
How about:
```suggestion
    <Text v-for="(responsible, index) in responsibles"
      >{{ responsible }}<template v-if="index + 1 < responsibles.length">, </template></Text
    >
```
To me it seems that css should be the technology to format the text, and not silly code formatting.

Reply:
Your suggestion is converted to my suggestion by eslint 😄